pkg/util/httprequest/httprequest_test.go

```diff
@@ -156,7 +135,7 @@ func TestSchemeHost(t *testing.T) {
 				},
 			},
 			expectedScheme: "http",
-			expectedHost:   "route-namespace.router.default.svc.cluster.local:80",
```

Comment:
`X-Forwarded-Host` is set to the hostname without a port, as it written in the test.
If you use `docker pull route:80/foo/bar,` then it'll be with the port.